Cairo Desktop/Cairo Desktop/SystemHotKeys.cs

```diff
@@ -0,0 +1,77 @@
+﻿using CairoDesktop.Common;
+using System;
+using System.Runtime.InteropServices;
+using System.Windows.Input;
+
+namespace CairoDesktop
+{
+    internal class SystemHotKeys
+    {
+        internal static void RegisterSystemHotkeys()
+        {
+            new HotKey(Key.R, KeyModifier.Win | KeyModifier.NoRepeat, OnWinRCommand);
```

Comment:
These don't seem to trigger for me due to the WinKey press down opening the Programs menu. Is that currently expected?

Reply:
I actually receive a quick flash of the programs menu then the run dialog

Reply:
This seems to be a bug in the HotKey system itself.

Reply:
I wonder if we could handle the "windows key only" case better so that it isnt triggered if another key is pressed.

Reply:
Windows only may be handled on release if no other keys were pressed. I’ll investigate more and try to alter if possible

Reply:
Ill have to brainstorm a bit but I think I will have to add a lowlevelkeyboard monitor to our HotKey system that will watch for modifier key down and ups. When a key-down is encountered it should set a flag that gets flipped if another hotkey is raised. If the flag is still set on release that would mean that no other hotkey was encountered and the modifier only hot key should get executed.

We can leave the code as is or comment it out until it gets altered.